src/test/feature/lib/hdfs_config.h

```diff
@@ -0,0 +1,172 @@
+#ifndef HAWQ_SRC_TEST_FEATURE_LIB_HDFS_CONFIG_H_
+#define HAWQ_SRC_TEST_FEATURE_LIB_HDFS_CONFIG_H_
+
+#include <string>
+#include <vector>
+
+#include "psql.h"
+#include "sql_util.h"
+#include "xml_parser.h"
+
+namespace hawq {
+namespace test {
+
+/**
+ * HdfsConfig common libray. Get detailed information about HDFS
+ * including checking state of namenodes and datanodes, get parameter value
+ * @author Chunling Wang
+ */
+class HdfsConfig {
+  public:
+    /**
+     * HdfsConfig constructor
+     */
+    HdfsConfig(): psql(HAWQ_DB, HAWQ_HOST, HAWQ_PORT, HAWQ_USER, HAWQ_PASSWORD) {}
+
+    /**
+     * HdfsConfig desstructor
+     */
+    ~HdfsConfig() {}
+
+    /**
+     * whether HDFS is in HA mode
+     * @return true if HDFS is HA
+     */
+    bool isHA();
+
+    /**
+     * whether HDFS is kerbos
+     * @return true if HDFS is kerbos
+     */
+    bool isKerbos();
```

Comment:
I am not sure kerbos is a short name of kerberos. I have googled that, I think you use a incorrect library name.